src/content/decentralized-identity.md

```diff
@@ -0,0 +1,157 @@
+
+### 2. KYC Authentication {#kyc-authentication}
+Using many online services requires individuals to provide attestations and credentials, such as a driving license or national passport. But this approach is problematic because private user information can be compromised and service providers cannot verify the authenticity of the attestation. 
+
+Decentralized identity allows companies to skip on conventional Know-Your-Customer (KYC) processes and authenticate user identities via Verifiable Credentials. This reduces the cost of identity management and prevents the use of fake documentation. 
+
+### 3. Voting and online communities {#voting-and-online-communities}
+
+Online voting and social media are two novel applications for decentralized identity. Online voting schemes are susceptible to manipulation, especially if malicious actors create false identities to vote. Asking individuals to present on-chain attestations can improve the integrity of online voting processes. 
+
+Decentralized identity can enable the creation of online communities that are free of fake accounts. Each user must authenticate their identity using an on-chain identity system, like the Ethereum Name Service, reducing the possibility of bots overrunning social platforms. 
+
+## Use decentralized identity applications on Ethereum
+
+There are also many ambitious projects using Ethereum as a foundation for decentralized identity solutions:
+
+**Ethereum Name Service (ENS) - _A decentralized naming system for on-chain, machine-readable identifiers, like, Ethereum wallet addresses, content hashes, and metadata._**
+- [Website](https://ens.domains/)
+
+**SpruceID - _A decentralized identity project which allows users to control digital identity with Ethereum accounts and ENS profiles instead of relying on third-party services._** 
+- [Website](https://www.spruceid.com/)
+
+**Proof of Humanity - _Proof of Humanity (or PoH) is a social identity verification system built on Ethereum._**
+- [Website](https://www.proofofhumanity.id/)
+
+**BrightID - _A decentralized, open-source social identity network seeking to reform identity verification through the creation and analysis of a social graph._**
+- [Website](https://www.brightid.org/)
+
+## Further reading
+
+- [Blockchain Use Cases: Blockchain in Digital Identity](https://consensys.net/blockchain-use-cases/digital-identity/) by ConsenSys
+- [What is Ethereum ERC725? Self-Sovereign Identity Management on the Blockchain](https://cryptoslate.com/what-is-erc725-self-sovereign-identity-management-on-the-blockchain/) by Sam Town
+- [How Blockchain Could Solve the Problem of Digital Identity](https://time.com/6142810/proof-of-humanity/) by Andrew R. Chow
+- [What Is Decentralized Identity And Why Should You Care?](https://web3.hashnode.com/what-is-decentralized-identity) by Emmanuel Awosika
```

Comment:
This should follow the same convention as our other use case pages. [e.g. see our DeFi page](https://ethereum.org/en/defi/#futher-reading)

Reply:
Alright. I'll check for some resources I can add here.

Reply:
Agree, but wouldn't call this a blocker personally. If we can find a few more things that would be great.

Reply:
I got some resources that could be useful and added them. LMK if they are good to go.